src/serializers/fields.rs

```diff
@@ -200,7 +200,15 @@ impl GeneralFieldsSerializer {
                     };
                     output_dict.set_item(key, value)?;
                 } else if field_extra.check == SerCheck::Strict {
-                    return Err(PydanticSerializationUnexpectedValue::new_err(None));
+                    let type_name = field_extra.model_type_name();
+                    return Err(PydanticSerializationUnexpectedValue::new_err(Some(format!(
+                        "Unexpected field `{key}`{for_type_name}",
```

Comment:
This is a drive-by improvement to the warning which was necessary for me to debug.

Reply:
Reminds me of https://github.com/pydantic/pydantic-core/issues/1483

Reply:
Nice improvement though, thanks

Reply:
Should we go ahead and fix #1483 while we're at it here?

Reply:
Last comment here - we should do a quick search through the codebase to see if other (similar) warnings can be updated with this structure.

Reply:
I think I will punt on both of these and leave them for a later refactoring of serialization warnings.